tests/tests_pytorch/trainer/optimization/test_manual_optimization.py

```diff
@@ -305,7 +305,8 @@ def on_train_epoch_end(self, *_, **__):
 
 
 @RunIf(min_cuda_gpus=1)
-def test_multiple_optimizers_step(tmp_path):
+@pytest.mark.parametrize("dicttype", [dict, collections.OrderedDict])
```

Comment:
can we test something other than collections.OrderedDict? depending on the python version that's the default dict implementations as at some point they made every dict to be ordered :D

Reply:
done, added a custom mapping (which just uses dict internally)